h2o-algos/src/test/java/hex/glm/GLMBasicTestRegression.java

```diff
@@ -145,7 +145,7 @@ public void testSparse() {
     parms._train = _weighted._key;
     parms._ignored_columns = new String[]{_weighted.name(0)};
     parms._response_column = _weighted.name(1);
-    parms._standardize = true;
+    parms._standardize = false;
```

Comment:
@wendycwong changing  this did not break a test. Actually I was changing  as many standardize properties as possible without breaking tests....to show how strange it is.